slices/slices.go

```diff
@@ -128,6 +128,12 @@ func Contains[E comparable](s []E, v E) bool {
 	return Index(s, v) >= 0
 }
 
+// ContainsFunc reports whether at least one element in s
+// satisfies f(s[*]).
+func ContainsFunc[E comparable](s []E, f func(E) bool) bool {
```

Comment:
This should be `E any` to match IndexFunc, no?

Reply:
You are absolutely correct, thank you!